lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+require './lib/twilio_sms'
+require './lib/telinfo'
+
+class Takeaway
+  attr_reader :order
+  include TelInfo
+
+  def initialize(takeaway_phone = Telephone.new)
+    @takeaway_phone = takeaway_phone
+  end
+
+  def menu
+    {
+      spring_rolls: 4.55,
+      grilled_dumpling: 4.9,
+      chicken_fried_rice: 6.95,
+      won_tom: 7.95,
+      curry_breef_brisket: 7.95,
+      coke: 1.2,
+      ice_tea: 2.55
+    }
+  end
+
+  def take_order(order, customertel = CUSTOMERTEL)
+    @order = order
+    validate_dishes
+    text = "Thank you! Your order was placed and will be delivered before"
+    body = "#{text}  #{(Time.now + 3600).strftime('%H:%M')}"
+    @takeaway_phone.send_sms(customertel, body)
+  end
+
+  def order_match?(num)
+    order.values.reduce(& :+) == num
```

Comment:
This method currently matches the quantity ordered, not the total price. Is this useful to the customer?